src/components/PrimaryPanes/SourcesTree.js

```diff
@@ -57,17 +51,17 @@ import type { SourcesMap } from "../../reducers/types";
 import type { Item } from "../shared/ManagedTree";
 
 type Props = {
-  selectSource: string => void,
-  setExpandedState: (Set<string>) => void,
-  setProjectDirectoryRoot: string => void,
-  clearProjectDirectoryRoot: void => void,
   sources: SourcesMap,
   sourceCount: number,
   shownSource?: string,
   selectedSource?: Source,
   debuggeeUrl: string,
   projectRoot: string,
-  expanded?: boolean
+  expanded?: boolean,
+  selectSource: string => *,
+  setExpandedState: (Set<string>) => *,
+  setProjectDirectoryRoot: string => *,
+  clearProjectDirectoryRoot: void => *
```

Comment:
we dont care about the return type

Reply:
then it's better to use "any" or "mixed" than "*", if you don't care. With "*" you tell Flow to infer it for you from what is passed, which could be an issue if you have several instances of the component where you pass different functions returning different things.
I'd suggest "mixed".

Reply:
okay, makes sense